### PR TITLE
Doc backends

### PR DESCRIPTION
**Pull request by tacaswell**
There is alot more work needed on this section, but is an improvement.

Closes https://github.com/matplotlib/jupyter-matplotlib/issues/13

### CODE REVIEW

File: tutorials/introductory/usage.py [hunk 1]
```diff
@@ -452,7 +465,7 @@ def my_plotter(ax, data1, data2, param_dict):
 # .. _PyQt5: https://riverbankcomputing.com/software/pyqt/intro
 #
 # WX backends
-# ===========
+# -----------
 #
 # At present the release version of `wxPython` (also known as wxPython classic)
 # does not support python3. A work in progress redesigned version known as
```
Comment (WeatherGod) [inline tutorials/introductory/usage.py]: Is this statement still true? wxpython 4.0.0b1 seems to support py3k. I don't know when the support started, though.

Comment (jenshnielsen) [reply tutorials/introductory/usage.py]: wxpython 4.0 is the same as Phonix which is the first version to support python 3. We could update this to point to wx 4.0 beta

**Comment by tacaswell** [comment]
I got rid of the stand-alone paragraph and just noted that v4 is required.

File: tutorials/introductory/usage.py [hunk 2]
```diff
@@ -413,27 +412,42 @@ def my_plotter(ax, data1, data2, param_dict):
 # ============   ================================================================
 # Backend        Description
 # ============   ================================================================
-# GTKAgg         Agg rendering to a :term:`GTK` 2.x canvas (requires PyGTK_ and
-#                pycairo_ or cairocffi_; Python2 only)
+# Qt5Agg         Agg rendering in a :term:`Qt5` canvas (requires PyQt5_).  This
+#                backend can be activated in IPython with ``%matplotlib qt5``.
+# ipympl         Agg rendering embedded in a Jupyter widget.  This can be enabled
```
Comment (QuLogic) [inline tutorials/introductory/usage.py]: ? This doesn't come with Matplotlib; should there be some reference for installing it (like the other requirements)?

Comment (tacaswell) [reply tutorials/introductory/usage.py]: Yes, PR shortly.

Comment (tacaswell) [reply tutorials/introductory/usage.py]: xref #9126

**Pull request merged**